tensorrt_llm/_torch/modules/mamba/mamba2_mixer.py

```diff
@@ -51,6 +51,7 @@ def __init__(
         apply_silu: bool = True,
         rms_norm_eps: float = 1e-5,
         dtype: Optional[torch.dtype] = None,
+        mamba_ssm_cache_dtype: Optional[torch.dtype] = None,
```

Comment:
I think this is unused?

Reply:
Also, since the mamba ssm cache type will not change during model lifetime, do you think it makes sense to actually send it to the `Mamba2Mixer` ctor and save it as an attribute? That way we don't need to access the kv cache manager in every forward pass just to get the ssm cache dtype